.github/workflows/tests.yml

```diff
@@ -1,6 +1,13 @@
 name: CI
 env:
-  COVERALLS_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+  PROJECT_ROOT: /root/go/src/github.com/infrawatch/sg-core
+
+  LOKI_IMAGE: grafana/loki:2.1.0
```

Comment:
For version tags, might be ideal to put these all in one spot and refer to them via env vars just to make it easier to update this in the future.

Reply:
Disregard this one. Looks fine.